ci/build_stf.yml

```diff
@@ -0,0 +1,46 @@
+---
+- name: "Do a local build of STF"
+  hosts: controller
+  tasks:
+    - name: "Set the value of sto_dir"
+      ansible.builtin.set_fact:
+        sto_dir: '{{ ansible_env.HOME }}/{{ zuul.project.src_dir }}'
+
+    - name: "Check for scripts"
+      ansible.builtin.stat:
+        path: "{{ sto_dir }}/build"
+      register: stat
+
+    - name: "Show the contents of {{ sto_dir }}"
+      ansible.builtin.debug:
+        var: stat
+
+    - name: "Get registry info"
+      ansible.builtin.shell:
+        cmd: |
+          oc registry info --internal
+      register: registry
+
+    - name: "Show the registry information"
+      ansible.builtin.debug:
+        var: registry.stdout
+
+    - name: "Build STF containers"
+      ansible.builtin.import_role:
+        name: '../build/stf-run-ci'
+      vars:
+        __deploy_stf: true
+        __local_build_enabled: true
+        __deploy_from_index_enabled: true
+          #__deploy_from_bundles_enables: false
+        __service_telemetry_snmptraps_enabled: true
+        __service_telemetry_storage_ephemeral_enabled: true
+        __working_branch: master
+        base_dir: "{{ sto_dir }}/build"
+        __service_telemetry_observability_strategy: "use_redhat"
+        # Is this the one to use for CRC?
+        # __internal_registry_path: "default-route-openshift-image-registry.apps-crc.testing"
+        __internal_registry_path: "{{ registry.stdout }}"
+        # TODO: These should be documented at some point
+        setup_bundle_registry_auth: false
+        setup_bundle_registry_tls_ca: false
```

Comment:
If these get added during this PR, then prior to merge the README.md should be updated.

Reply:
They got added in one of the changes @vkmc made. README was updated then too.